lamindb/_can_curate.py

```diff
@@ -59,6 +59,7 @@ def inspect(
     field: str | StrField | None = None,
     *,
     mute: bool = False,
+    strict: bool = False,
```

Comment:
Are we sure we want to introduce yet another parameter for the user?

If so, please craft the docstring for it and provide an example in the docs.

Reply:
The term `strict` is also potentially overloaded and a more specific name could be adequate; see e.g. the meaning in the pandera validation API: https://pandera.readthedocs.io/en/stable/dataframe_schemas.html

Reply:
No, it's just for backward compatibility with downstream packages behaviors now.

Reply:
I'm not adding to the docstring because I don't want users to use it yet, and I hope this is never needed.

Reply:
But isn't this a user-facing method signature? If it is, it should be documented. If it shouldn't be user-facing, how about pre-fixing it with an underscore and/or making it part of kwargs?